lethal-squirrels/Bangazon-Site
# Ben order history

## REFERENCE

Ticket: #16 
## DESCRIPTION
Made an OrderHistoryViewModel, View, and associated method in the Profile Controller.
When you go to Profile - Order History, it displays the user's closed orders.
Made an OrderDetailsViewModel, View, and associated method in the Profile Controller.
When you click "Details" on an order on the Order History page, it takes you to a page displaying the details of that view, including the dat, orderID, order total, and a table of the line items on that order.

## STEPS TO TEST
1. Clone this branch to a local repo
2. In the package manager, run `update-database`. Make sure you have seed data. Reseed the DB if not.
3. Test functionality
	* Go to Order History
	* Click "Details" to see an orders details.
	* Try with at least 2 users to make sure it works in different cases. Try break it
